package.json

```diff
@@ -94,8 +94,8 @@
     "yarn-audit-fix": "^9.3.2"
   },
   "peerDependencies": {
-    "@nestjs/common": "^7.0.9 || ^8.0.0",
-    "@nestjs/core": "^7.0.9 || ^8.0.0",
+    "@nestjs/common": "^7.0.9 || ^8.0.0 || ^9.0.0",
+    "@nestjs/core": "^7.0.9 || ^8.0.0 || ^9.0.0",
```

Comment:
Please add:
```json
"@nestjs/common": "^7.0.9 || ^8.0.0 || ^9.0.0 || ^9.0.4",
"@nestjs/core": "^7.0.9 || ^8.0.0 || ^9.0.0 || ^9.0.4",
```

Reply:
> **Note:** Already create a [PR](https://github.com/nest-modules/mailer/pull/818

Reply:
I am pretty positive this is not necessary. ^9.0.0 already includes 9.0.4.

https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept

Reply:
I try use this package with version 9.0.4 and not work.

Reply:
![image](https://user-images.githubusercontent.com/18479474/179848736-71b209b9-61c0-4857-8c8a-ea8a0bf5ef29.png)

Reply:
That is because the author has accepted this MR, but has not published a release to npmjs.org yet. You should open a new ticket asking @juandav to publish [1.8.0](https://github.com/nest-modules/mailer/commit/414cd06f7e394f9d13b777ae4adb7fec8179fac4) to npmjs.org

https://www.npmjs.com/package/@nestjs-modules/mailer